webapp/templates/_ga.html

```diff
@@ -5,5 +5,18 @@
 })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
 ga('create', 'UA-89387706-1', 'auto');
+// self-navigator.js sends more pageview events for in-page navigations.
 ga('send', 'pageview');
+
+window.addEventListener('error', function(message, source, lineno, colno) {
```

Comment:
```suggestion
window.onerror = function(message, source, lineno, colno) {
```

FYI @stephenmcgruer : it turns out I didn't read MDN closely enough. The callback for `addEventListener` only takes a single argument `event`, the form of callback I'm using is for `window.onerror`.

Reply:
And thus turns out that I did not review well enough, sorry :(.

Still LGTM.